plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -704,7 +704,7 @@ public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle ha
         }
     }
 
-    protected String buildInsertSql(ConnectorSession session, RemoteTableName targetTable, RemoteTableName sourceTable, List<String> columnNames)
+    protected String buildInsertSql(RemoteTableName targetTable, RemoteTableName sourceTable, List<String> columnNames)
```

Comment:
Session parameter could be used other connectors that are using trino-base-jdbc library but they are not contributed to Trino project.

Notice that trion-base-jdbc is library and changing API is not trivial. Almost like changing API in SPI.